source/release-notes/backend.txt

```diff
@@ -13,6 +13,22 @@ Atlas App Services Changelog
 2023 Releases
 -------------
 
+.. _backend_20230315:
+
+15 March 2023 Release
+~~~~~~~~~~~~~~~~~~~~~
+
+- Expiration times for user refresh tokens can be :ref:`configured from the UI
+  <TODO: DOCSP-27081>`.
+- Change the email address associated with email/password authentication via an
+  :admin-api-endpoint:`Admin API endpoint <operation/adminResetUserEmail>`
+- :ref:`Device Sync is automatically paused <mongodb-service-limitations-sync-inactive-pause>`
+  if it is inactive for 30 days.
+- Updated all existing apps using Atlas Device Sync with Flexible Sync to
```

Comment:
I have one minor question about wording for this one. Is this effectively an automatic migration to a new app/config structure? I wonder if we want to link to [the "Unified Rules System" section](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-28553/rules/sync-compatibility/#unified-rules-system) on that page, which goes into detail about the changes, and be more explicit that "Updated all existing apps" was some sort of automatic migration.

Reply:
sounds good to me. didn't know that page existed 😄